csi.proto

```diff
@@ -350,7 +355,8 @@ message CreateVolumeRequest {
   // This field is REQUIRED.
   repeated VolumeCapability volume_capabilities = 3;
 
-  // Plugin specific parameters passed in as opaque key-value pairs.
+  // Plugin specific parameters passed in as opaque key-value pairs 
+  // for provision time.
   // This field is OPTIONAL. The Plugin is responsible for parsing and
   // validating these parameters. COs will treat these as opaque.
```

Comment:
// This field is OPTIONAL. Creation-time opaque parameters to be
// parsed and validated by the Plugin. COs will treat these as opaque.